pynicotine/config.py

```diff
@@ -315,10 +315,10 @@ def load_config(self):
                 "filtercc": [],
                 "filterin": [],
                 "filterout": [],
-                "filtersize": [],
-                "filterbr": [],
-                "filtertype": [],
-                "filterlength": [],
+                "filtersize": [">50MiB", ">20MiB|<50MiB", ">10MiB|<20MiB", ">5MiB|<10MiB", "<=5MiB"],
+                "filterbr": ["!0", "128|<224", ">192|<=320", "=320|=1411", ">320"],
+                "filtertype": ["flac|wav|aiff|cue|iso", "mp3|m4a|aac|ogg|opus", "mp4|m4v|mkv|mov|264", "!mp3|!wma]"],
```

Comment:
Could you add WavPack back to the first (lossless) filter? Not sure about including iso here, is it used for anything other than commercial SACDs?

Reply:
Shouldn't we remove pipes from these default filters?